app.js

```diff
@@ -16,25 +16,25 @@ var app = express();
 app.set('views', path.join(__dirname, 'views'));
 app.set('view engine', 'jade');
 
-// uncomment after placing your favicon in /public
-//app.use(favicon(path.join(__dirname, 'public', 'favicon.ico')));
+
+
 app.use(logger('dev'));
 app.use(bodyParser.json());
-app.use(bodyParser.urlencoded({ extended: false }));
+app.use(bodyParser.urlencoded({extended: false}));
 app.use(cookieParser());
 app.use(session({
   secret: 'secret',
   name: 'browser-calls',
-  cookie: { maxAge: 60000 },
+  cookie: {maxAge: 60000},
   resave: true,
-  saveUninitialized: true
+  saveUninitialized: true,
```

Comment:
That's an interesting reason @joliveros, is about having a clean diff. If you have to add or remove a line in an object literal, the _diff_ will only contain the line you add or remove, e.g.:

When removing a line:

```diff
{
   foo: 'foo',
-  bar: 'bar',
 }
```

When adding a line:

```diff
 {
   foo: 'foo',
   bar: 'bar',
+  tar: 'tar',
 }
```

The same example when you don't have a trailing comma in the last element of your literal object and you want to add a new key.

```diff
 {
   foo: 'foo',
-  bar: 'bar'
+  bar: 'bar',
+  tar: 'tar'
 }
```

😉